cmd/airflow.go

```diff
@@ -529,24 +535,39 @@ func airflowInit(cmd *cobra.Command, args []string) error {
 	defaultImageName := airflow.AstroRuntimeImageName
 	if useAstronomerCertified {
 		defaultImageName = airflow.AstronomerCertifiedImageName
-		fmt.Printf("Initializing Astro project\nPulling Airflow development files from Astronomer Certified Airflow Version %s\n", defaultImageTag)
-	} else {
-		fmt.Printf("Initializing Astro project\nPulling Airflow development files from Astro Runtime %s\n", defaultImageTag)
+	}
+
+	projectDirExists, err := fileutil.Exists(config.WorkingPath, nil)
+	if err != nil {
+		return err
+	}
+	if !projectDirExists {
+		err := os.Mkdir(config.WorkingPath, 0755)
+		if err != nil {
+			return err
+		}
+	}
+	err = os.Chdir(config.WorkingPath)
+	if err != nil {
+		return err
 	}
 
 	emptyDir := fileutil.IsEmptyDir(config.WorkingPath)
 
 	if !emptyDir {
 		i, _ := input.Confirm(
-			fmt.Sprintf("%s \nYou are not in an empty directory. Are you sure you want to initialize a project?", config.WorkingPath))
+			fmt.Sprintf("%s is not an empty directory. Are you sure you want to initialize a project here?", config.WorkingPath))
 
 		if !i {
 			fmt.Println("Canceling project initialization...")
 			return nil
 		}
 	}
 
-	exists := config.ProjectConfigExists()
+	exists, err := config.IsProjectDir(config.WorkingPath)
+	if err != nil {
+		return err
+	}
```

Comment:
Alternative function we already had to check if the project config file exists in the directory or not. The prior version assumes that the process is already within the working directory, where the alternative lets us check if the file exists at the certain path.